src/jose/algorithms/signing/secp256k1.ts

```diff
@@ -57,7 +57,7 @@ export const secp256k1: Signer = {
     validateKey(publicJwk);
 
     const xBytes = Encoder.base64UrlToBytes(publicJwk.x);
-    const yBytes = Encoder.base64UrlToBytes(publicJwk.y);
+    const yBytes = Encoder.base64UrlToBytes(publicJwk.y!);
```

Comment:
Don't think it's safe to say `.y` is guaranteed here
```suggestion
    let yBytes = new Uint8Array([]);
    if (publicJwk?.y !== undefined) {
      yBytes = Encoder.base64UrlToBytes(publicJwk.y);
    }
```

Reply:
Done.